src/Table.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import allCountryScores from "./scores";
+import "./Table.css"
+
+function Table() {
+  return (
+    <>
+      <h1>High score per Country</h1>
+      <div>
+        {allCountryScores.map(({ name, scores }, index) => {
+          return (
+            <div key={index}>
+              <table>
+                <thead>
+                  <tr>
+                    <th scope="col">{name}</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  {scores.map(({ n, s }, index) => {
```

Comment:
Great use of maps! I think we can use some more descriptive variable names here however, which introduces a little destructuring trick.

Personally I don't like the variable names `n` and `s`, so if I want to, I can rename `n` as `personName` and `s` as `score` using this syntax:

```js
{scores.map(({n : personName, s : score }, index) => {
    // map logic here
    console.log(`A person with the name '${personName}' has a score of ${score}`)
})}
```

Reply:
I will keep this in mind for future